<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingDataSelector</class>
 <widget class="QWidget" name="TrainingDataSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QPushButton" name="btn_add_file">
     <property name="text">
      <string>Add File</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btn_clear">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="listView_files"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="btn_generate_model">
     <property name="text">
      <string>Generate Model</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_show_task_name">
     <property name="text">
      <string> </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
